config/action.d/firewallcmd-rich-rules_new.conf

```diff
@@ -0,0 +1,58 @@
+# Fail2Ban configuration file
```

Comment:
If it gets deprecated, why not update the original action instead?

Reply:
Hello! I've done it. Please see.

Reply:
Hmm...
Now almost everything got changed in firewallcmd-actions.
Unexpected to solve few "deprecations".

Reply:
That was my first thought. That's why I have made separate file at first. So what's the right way to go to make it merged?

Reply:
@sebres Hello! Please tell me what's the right way you want me to do? I would like to make it merged.